src/service/viewport-impl.js

```diff
@@ -134,6 +134,10 @@ export class Viewport {
 
     this.binding_.onScroll(this.scroll_.bind(this));
     this.binding_.onResize(this.resize_.bind(this));
+
+    this.onScroll(() => {
+      this.viewer_.postScroll(this.getScrollTop());
```

Comment:
I think we should consider only sending scroll event once per animation frame. @cramforce WDYT?